tools/build/libwebrtc/config.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#=============================================================================
+# Global config
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#-----------------------------------------------------------------------------
+function check-err() {
+    rv=$?
+    [[ $rv != 0 ]] && echo "$0 exited with code $rv. Run with -v to get more info."
+    exit $rv
+}
+
+trap "check-err" INT TERM EXIT
+
+#-----------------------------------------------------------------------------
+
+BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "$BUILD_DIR/lib.sh"
+
+#=============================================================================
+# Functions
+
+#-----------------------------------------------------------------------------
+function usage() {
```

Comment:
I've just spent 4 hours setting up an Ubuntu VM with Hyper-V (╯°□°）╯︵ ┻━┻ and resizing the virtual disk to accommodate the gigantic repository (currently at 37GB total partition size) (╯°□°）╯︵ ┻━┻, to be told by the Google setup script that:
```
ERROR: The only supported distros are
 	Ubuntu 14.04 LTS (trusty)
 	Ubuntu 16.04 LTS (xenial)
 	Ubuntu 17.10 (artful)
 	Ubuntu 18.04 LTS (bionic)
 	Debian 8 (jessie) or later
``` 
Yes, I used Ubuntu 19.10 as suggested by default by Hyper-V. 😡

So I really think we should test ASAP in those scripts, preferably during `config.sh`:
1. The user runs a valid distribution (see list above)
2. The user has at least 40GB of disk space available

The actual check is in [`webrtc/build/install-build-deps.sh`](https://chromium.googlesource.com/chromium/src/+/master/build/install-build-deps.sh#110) which is outdated on `branch-heads/71` compared to recent Ubuntu releases (as linked on `master` it actually supports `eoan` 19.10). Ideally we'd grab that file and extract the supported distros from it, but it might be a bit complex to reach it via the chain of dependency (git branch => sub-repo chromium/build => script => grep on `supported_codenames`).

Reply:
That is terrible! Yes, we need to verify platform earlier.

Reply:
I pushed some basic checks as discussed (see d4255487ea4217ce1545d0d5950cf508e8580b7a). Can you please have a look @eanders-MS? My bash is quite rusty... :wink: